### DISCUSSION TITLE
他妈的，绳网怎么成了这么个样子！

### DISCUSSION THREAD

**Discussion by Larksonguesinaspic** [share121/inter-knot#970] [其他]
![image](https://github.com/user-attachments/assets/c979d248-f495-4240-98a0-4b8659215bd1)
(只是做题做累了发个癫)

**Comment by nostalgiatan** [comment]
> ![image](https://github.com/user-attachments/assets/c979d248-f495-4240-98a0-4b8659215bd1)
> (只是做题做累了发个癫)

最好还是不要带脏字了。

**Comment by 1451374815** [comment]
别怕，绳网可能会倒闭，但永远不会变质

**Comment by Wu-jian-NGX-zhanghao** [comment]
同意楼上

**Comment by Mayf1ower** [comment]
你说得对，xx可能会xx,但永远不会xx

**Comment by Qiling-XwX** [comment]
![38B2843718D7F614ACB3B77A54871523](https://github.com/user-attachments/assets/459e8695-5cc0-4056-9dbb-ecc59f13b8ed)

**Reply by asderfgh** [reply]
可爱

**Comment by Joker-JYH** [comment]
跟一群虫豸在一起，怎么做的好绳斤呢？
